src/main/java/core/basesyntax/model/Transaktion.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+public class Transaktion {
```

Comment:
```suggestion
public class Transaction {
```
change it everywhere :)